ts/webui/src/components/Overview.tsx

```diff
@@ -167,6 +166,13 @@ class Overview extends React.Component<{}, OverviewState> {
                                             </Stack>
                                         </div>
                                     </Stack>
+                                    <div className='overviewChart'>
+                                        <Accuracy
+                                            accuracyData={accuracyGraphData}
+                                            accNodata={noDataMessage}
+                                            height={290}
```

Comment:
Is it possible to move the styles into the css file?

Reply:
ok